Jenkinsfile

```diff
@@ -3202,7 +3204,6 @@ assert_frame_equal(training_curve, gt_curve, rtol=1e-3, atol=1e-3)"'''
         trainer.accelerator=gpu \
         trainer.log_every_n_steps=1 \
         trainer.val_check_interval=2 \
-        trainer.limit_val_batches=1 \
```

Comment:
@aklife97 thanks, adding it back in. During the migration to 2.0 there was an issue using this and hence was removed temporarily. Missed to add it back.

Reply:
@aklife97 sorry my bad.Actually this trainer.limit_val_batches=1 needs to be removed or made as 2 (which is the num of val batches). Otherwise, there's an error with the datalaoder_iter.

Reply:
setting it to 2 sounds good, can we also change other tests that remove it?